enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -432,15 +432,21 @@ func ensureDefaultPartitionExists(opts *Options) {
 // defaultPebbleOptions returns default pebble config options.
 func defaultPebbleOptions(el *pebbleEventListener) *pebble.Options {
 	// These values Borrowed from CockroachDB.
-	return &pebble.Options{
-		MaxConcurrentCompactions: 6,
+	opts := &pebble.Options{
+		L0CompactionThreshold:    2,
+		MaxConcurrentCompactions: 12,
 		MemTableSize:             64 << 20, // 64 MB
 		EventListener: pebble.EventListener{
 			WriteStallBegin: el.WriteStallBegin,
 			WriteStallEnd:   el.WriteStallEnd,
 			DiskSlow:        el.DiskSlow,
 		},
 	}
+
+	opts.Experimental.L0CompactionConcurrency = 2
```

Comment:
```suggestion
	// The threshold of L0 read-amplification at which compaction concurrency
	// is enabled (if CompactionDebtConcurrency was not already exceeded).
	// Every multiple of this value enables another concurrent
	// compaction up to MaxConcurrentCompactions.
	opts.Experimental.L0CompactionConcurrency = 2
```

Reply:
this is 2 and L0CompactionThreshold is 2, so parallel compactions will start immediately. We could make this 4 or something for a more gradual ramp up?